chargeio/infrastructure/Requestor.cs

```diff
@@ -39,12 +37,12 @@ public static string PutJson(string url, string putData, string secretKey = null
 
         public static string Delete(string url, string secretKey = null)
         {
-            return Delete(url).Result;
+            return _Delete(url, secretKey).Result;

```

Comment:
In every other case the request function is <HTTP Verb>+<expected return type>. In the case of the Delete function there is no return type so the internal Delete request function clashes with the public interface method name. The _ prefix is common for internal .NET methods.

Reply:
The other functions are actually HTTP verb + accepted content type, not return type. I see your point with the signature clash, but rather than create an inner _Delete() that is only ever called by the public Delete() with the exact same signature, we should just move the body of _Delete() into Delete() and remove the extraneous method.